storage/xtradb/handler/ha_innodb.cc

```diff
@@ -8681,23 +8681,21 @@ ha_innobase::write_row(
 
 	sql_command = thd_sql_command(user_thd);
 
+	bool is_load_data_split= false;
```

Comment:
Please do not make changes to xtradb as it is newer used or compiled on >=10.2.

Reply:
this is added only for compatibility with changes in the main innodb branch - otherwise xtradb will behave incompatible with innodb

Reply:
How do you compile XtraDB on 10.2? The `storage/xtradb` in 10.2 should not be modified except in merges. I would have removed the directory already in 10.2, because it is essentially dead code. It has been removed in 10.3.

Reply:
fixed, all changes in xtradb are removed from the patch